app/models/classification.rb

```diff
@@ -269,9 +269,7 @@ def add_entry(options)
     children.create!(options)
   end
 
-  def entries
-    children
-  end
+  alias entries children
```

Comment:
Sometimes you can't alias things, particularly when they dynamically come from another place.  If I recall, children comes from acts_as_tree.  If we have tests demonstrating that entries still works after this change, then I'm good with it (though I wonder if the alias should be moved up closer to the acts_as_tree definition (assuming it comes from acts_as_tree).

Reply:
I think if this were gonna fail on this change it would have done so on the existing tests: https://github.com/ManageIQ/manageiq/blob/master/spec/models/classification_spec.rb#L173, specially since we have a context that handles the hierarchy: https://github.com/ManageIQ/manageiq/blob/master/spec/models/classification_spec.rb#L55